app/src/ComponentsOverviewPage.tsx

```diff
@@ -26,29 +18,17 @@ import { TrackSelectionList } from '../../src/components/TrackSelectionList'
 import { VolumeBar } from '../../src/components/VolumeBar'
 import { PrestoContext, PrestoContextType } from '../../src/context/PrestoContext'
 
-import { Asset } from './Asset'
-
+import { PageProps } from './types'
 
 /**
  * An overview of available UI components that can be
  * assembled to create a skin.
  */
-export const ComponentsOverviewPage = (props: {
-  asset?: Asset
-  autoload?: boolean
-}) => {
+export const ComponentsOverviewPage = (props: PageProps) => {
   // We track the thumb position to showcase how we can manually load thumbs
   const [thumbPosition, setThumbPosition] = useState<number | undefined>()
   const [context, setContext] = useState<PrestoContextType|null>(null)
-
-  // Create the player as state of this component
-  const [player] = useState(new Player((pp: clpp.Player) => {
-    pp.use(clpp.dash.DashComponent)
-    pp.use(clpp.hls.HlsComponent)
-    pp.use(clpp.htmlcue.HtmlCueComponent)
-    pp.use(clpp.ttml.TtmlComponent)
-    pp.use(clpp.vtt.VttComponent)
-  }))
```

Comment:
I've replaced these calls by `clpp.install()` which is the preferred API and moved it to App.tsx to simplify things.